codespell_lib/data/dictionary.txt

```diff
@@ -21438,8 +21611,11 @@ envrironments->environments
 envrirons->environs
 envvironment->environment
 enxt->next
+enymore->anymore
 enything->anything
+enytime->anytime
 enyway->anyway
+enywhere->anywhere
 epecifica->especifica
 epect->expect
```

Comment:
Perhaps add `elect` and even `eject` and `erect` as fixes for `epect`?

Reply:
I found `elect` but not `eject` or `erect`. I'll add `elect` to #3049.